LOG.tbasicu

```diff
@@ -150,33 +155,32 @@ Type tLogThis
     Local sTagF as String
     Local sDT as String
     Local sDTF as String
-    Local successFile as Number
```

Comment:
This flag will not be needed, you will see.

Reply:
seen :)